docs/next.md

```diff
@@ -9,11 +9,13 @@ reduce the amount of magic and just rely on ansible core features.
 - `roles-path` and `collections-paths` are no longer configurable for
   dependencies. Users are expected to make use of `ansible.cfg` file to
   alter them when needed.
+- testinfra verifier driver was removed but current users should be able to
+  keep using using their testinfra tests if they call them using
+  `ansible.builint.shell` ansible modules from within `converge.yml` playbook.
```

Comment:
Thanks, I fixed the first two bullets. Regarding latest, at least initially v6 will not use any external drivers.

Please note that we do not want to prevent people from using testinfra, we only want to reduce the complexity of molecule and give testinfra users a fallback plan that does not require rewriting their testinfra tests. As the inventory location is available as an environment variable, it should be very easy to write a verify playbook that calls pytest the same way the plugin did. In addition, the user will be able to easily customize the arguments.